phpdotnet/phd/Package/Generic/XHTML.php

```diff
@@ -2040,6 +2046,119 @@ public function format_itemizedlist($open, $name, $attrs, $props) {
         return '</ul>';
     }
 
+    public function format_simplelist($open, $name, $attrs, $props) {
+        if ($open) {
+            $this->cchunk["simplelist"]["type"] = $attrs[Reader::XMLNS_DOCBOOK]["type"] ?? "";
+            $this->cchunk["simplelist"]["columns"] = $attrs[Reader::XMLNS_DOCBOOK]["columns"] ?? 1;
+
+            if ($this->cchunk["simplelist"]["columns"] < 1) {
+                $this->cchunk["simplelist"]["columns"] = 1;
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "inline") {
+                return '<span class="' . $name . '">';
+            }
+
+            if ($this->cchunk["simplelist"]["type"] === "vert"
+                || $this->cchunk["simplelist"]["type"] === "horiz") {
+                return '<table class="' . $name . '">' . "\n" . str_repeat(" ", $props["depth"] + 1) . "<tbody>\n";
```

Comment:
Are the str_repeat() calls for indentation? Maybe add a dedicated private method to make it clear what this call is for?